src/odemis/acq/stream/_base.py

```diff
@@ -959,6 +981,13 @@ def _updateHistogram(self, data=None):
 
         data = self.raw[0] if data is None else data
```

Comment:
Shouldn't there also be some checks like if self.raw and ifinstance(self.raw, list)? Like in line 902?

Reply:
This function already has this limitation. The idea is that the histogram is updated at the same time as the image. So it's also not needed to check in both functions, one error message is enough.